compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -30,7 +29,7 @@ final class Conversions(
 
   // The ledger data will not contain information from the partial transaction at this point.
   // We need the mapping for converting error message so we manually add it here.
-  private val ptxCoidToNodeId = ptx.nodes
+  private val ptxCoidToNodeId = incomplete.transaction.nodes
```

Comment:
@remyhaemmerle-da @nickchapman-da we use the nodes here at least. I guess we don’t use the roots so it works?  But that means we rely on having nodes not reachable by the roots which seems extremely confusing so I think fixing this should happen before merging this PR.

Reply:
This PR is a semantic no-op
Probably best to make the fix in a separate PR. To avoid confusing things.

Reply:
I don’t think it is. You’re transaction has different roots than the partial transaction we had before We get lucky because we don’t use the roots but I don’t think that’s a sensible state. You’re effectively building an invalid transaction with unreachable nodes while relying on those nodes being there.

Reply:
Are you sure?
The roots before were: `ptx.context.children.toImmArray.toSeq.sortBy(_.index)`
And they still are now.

Reply:
Offline we discuss and agree that this PR did not introduce any bug/change. But the original code from scenario-service which is now moved to `PartialTransaction.finishIncomplete` is somehow not correct and should be looked at in the future.